### ISSUE TITLE
Add images for all people who don’t currently have one

### ISSUE THREAD

**Issue by cattsmall** [QuantumBox/blackgamedevs#221]
A lot of people don’t have images right now. Can we add images from their games (ideal) or at least locate a photo?

**Comment by MaximumCrash** [comment]
@cattsmall from what I'm seeing with some responses and entries, some folks don't want to provide images for their entry on the site. Is that okay in regards to this issue?

**Comment by cattsmall** [comment]
Yup, that's okay! We can either use an image of their game OR keep it blank (as long as the placeholder shows up).
